website/docs/writing-docs.mdx

```diff
@@ -0,0 +1,176 @@
+---
+id: writing-docs
+title: Writing Docs
+---
+
+<!-- TODO: The title is a bit weird -->
+
+Next, let's touch on the powerful feature in Docusaurus - documentation.
+
+## Using Markdown
+
+Docusaurus supports Markdown with [Remark](https://github.com/remarkjs/remark) as the Markdown parser and is extensible via plugins.
```

Comment:
Since we're revising. If i'm not wrong it's not you who wrote this part. But I think there is some wrong info here (that i just noticed). 

Our markdown parser is MDX, under the hood it uses remark and rehype. But its misnomer to call it `Docusaurus supports Markdown with [Remark](https://github.com/remarkjs/remark)`

The full mdx specification can be seen here: https://github.com/mdx-js/specification

I think we should say its powered by mdx rather than remark

Reply:
You reminded me, I should have taken less granted of what is is already written out.